django/core/validators.py

```diff
@@ -238,6 +238,20 @@ def __eq__(self, other):
 
 validate_email = EmailValidator()
 
+
+@deconstructible
+class DomainNameValidator(RegexValidator):
+    message = _('Enter a valid domain name value.')
+    regex = _lazy_re_compile(
+        r'(?:' +
+        URLValidator.ipv4_re + '|' +
```

Comment:
Are the ip re's included by mistake? They aren't tested.

Reply:
IPs can be used in URL, so I guess that's what they mean here. Doesn't shock me.

Reply:
Hi, I was looking for a domain validator in Django and I've found this PR.

I think a domain name should also be checked against a max length, as `URLValidator` does (https://github.com/django/django/blob/master/django/core/validators.py#L137).